meta/categories.json

```diff
@@ -51,7 +51,7 @@
   },
   {
     "name": "Graphics & Design",
-    "count": 16,
```

Comment:
This is not the proper count to update ;)

Reply:
I think I'll just remove it then :+1:, It can cause all sorts of inconsistency depending on when this PR will be merged!